File: inflate/tests/langs/scala/Box.scala
package sample

class Box {
  private val size = 1

  def helper(value: Int): Int = {
    value + size
  }

  def greet(value: Int): Int = {
    helper(value) * 2
  }

  def runner(): Int = {
    greet(3)
  }
}

File: inflate/tests/langs/java/Box.java
package sample;

public class Box {
    private int size = 1;

    public int helper(int value) {
        return value + this.size;
    }

    public int greet(int value) {
        return helper(value) * 2;
    }

    public int runner() {
        return greet(3);
    }
}
